src/util/create-integrations-pages/constants.js

```diff
@@ -0,0 +1,18 @@
+const ADDON_FRAGMENT = `
+  id: name
```

Comment:
I was going to recommend this approach: https://www.gatsbyjs.com/docs/reference/graphql-data-layer/using-graphql-fragments/

But I think you can't do that here cause it's running in Node and not a page query?

Reply:
Yeah it didn't work in the gatsby-node context :(